TeamCode/src/main/java/org/firstinspires/ftc/teamcode/HardwareInnov8Robot.java

```diff
@@ -0,0 +1,127 @@
+package org.firstinspires.ftc.teamcode;
+
+import android.util.Log;
+
+import com.qualcomm.hardware.bosch.BNO055IMU;
+import com.qualcomm.hardware.bosch.JustLoggingAccelerationIntegrator;
+import com.qualcomm.robotcore.hardware.CRServo;
+import com.qualcomm.robotcore.hardware.ColorSensor;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorEx;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+import com.qualcomm.robotcore.hardware.Servo;
+import com.qualcomm.robotcore.hardware.TouchSensor;
+import com.qualcomm.robotcore.util.ElapsedTime;
+
+/**
+ * This is NOT an opmode.
+ * <p>
+ * This class can be used to define all the specific hardware for a single robot.
+ * In this case that robot is a Pushbot.
+ * See PushbotTeleopTank_Iterative and others classes starting with "Pushbot" for usage examples.
+ * <p>
+ * This hardware class assumes the following device names have been configured on the robot:
+ * Note:  All names are lower case and some have single spaces between words.
+ * <p>
+ * Motor channel:  Left  drive motor:        "left_drive"
+ * Motor channel:  Right drive motor:        "right_drive"
+ */
+public class HardwareInnov8Robot {
+    /* Public OpMode members. */
+    public DcMotor motorOne = null; // Front left wheel
+    public DcMotor motorTwo = null; // Back left  wheel
+    public DcMotor motorThree = null; // Front right wheel
+    public DcMotor motorFour = null; // Back right wheel
+
+
+    // Example for servos
+
+    //    public Servo
+
+
+    public BNO055IMU imu;
+//
+    // Examples for servos
+
+    public static final double MID_SERVO = 0.5;
+    public static final double START_SERVO = 0; // all the way down
+    public static final double END_SERVO = 1; // all the way up
+
+    /* local OpMode members. */
+    HardwareMap hwMap = null;
+    private ElapsedTime period = new ElapsedTime();
+
+    /* Constructor */
+    public HardwareInnov8Robot(HardwareMap ahwMap) {
+        this.hwMap = ahwMap;
+        this.init(ahwMap);
+
+    }
+
+    /* Initialize standard Hardware interfaces */
+    public void init(HardwareMap ahwMap) {
+        // Save reference to Hardware map
+
+
+        // Define and Initialize Motors
+        motorOne = this.hwMap.get(DcMotor.class, "motorOne");
+        motorTwo = this.hwMap.get(DcMotor.class, "motorTwo");
+        motorThree = this.hwMap.get(DcMotor.class, "motorThree");
+        motorFour = this.hwMap.get(DcMotor.class, "motorFour");
+        motorOne.setDirection(DcMotor.Direction.FORWARD); // Set to REVERSE if using AndyMark motors
```

Comment:
What kind of motors are you using in the code rn? If they are not andymark, we don't need to make comments about things that don't exist in out code. The information about what direction certain motors should be belongs in the engineering journal.
I would add a comment before you set the directions that says "using <XXX> motors"